scripts/zones/Balgas_Dais/mobs/Wyrm.lua

```diff
@@ -56,7 +68,7 @@ entity.onMobFight = function(mob, target)
         mob:lookAt(target:getPos())
     end
 
-    if mob:canUseAbilities() then
+    if mob:actionQueueEmpty() then
```

Comment:
Why use actionQueueEmpty over canUseAbilities?

Reply:
I believe the reasoning is that the action queue can be filled with a grounded TP move (such as dragon breath) and do very bad things if it changes into flying during the middle of it since it pivots to face who has hate in the air

Reply:
I personally think it would be best to combine actionQueueEmpty to also be checked in `canUseAbilities` so that check is all encompassing and leave all the Wyrm HNMs to still use that.

Reply:
All can use abilities does it check if its slept, impired, stunned, amnesia, petri, terror. Through testing both locally, beta and currently live there are 0 issues with any of this getting stuck or being broken.

Reply:
By using canUseAbilities it was using TP moves during phase change causing things to spike flail mid air. All that is fixed with he current proposed implementation

Reply:
Yeah what I meant was adjust canUseAbilities to check all of what you listed plus what actionQueueEmpty does.

Reply:
I wouldn't necessarily change how canUseAbilities works in its core functionality. I think it has its uses and a purpose elsewhere just not specifically for this fight. Is there a reason we need to check for this?